search-app/docs/api.md

```diff
@@ -1735,10 +1743,21 @@ Examples:
 
   curl -XPOST "%CMR-ENDPOINT%/collections" -F "shapefile=@box.kml;type=application/vnd.google-earth.kml+xml" -F "provider=PROV1"
 
-Internally a WGS 84 Coordinate Reference System (CRS) is used. The system will attempt to tranform shapefile geometry that uses a differnt CRS, but this is not guaranteed to work and the request will be rejected if a suitable tranformation is not found.
+Internally a WGS 84 Coordinate Reference System (CRS) is used. The system will attempt to transform shapefile geometry that uses a different CRS, but this is not guaranteed to work and the request will be rejected if a suitable transformation is not found.
 
 **NOTE:** This is an experimental feature and may not be enabled in all environments.
 
+#### <a name="c-shapefile-simplification"></a> Simplifying shapefiles during collection search
+Shapefiles are limited to 5000 points by deafult. A user can request that the CMR try to simplify (reduce the number of points) a shapefile by setting the `simplify-shapefile` parameter to `true` in the POST used to pass in the shapefile.
```

Comment:
Add some language such as "User using a shapefile larger than the CMR supported shapefile limits can use the `simplify-shapefile` parameter to ...." to make it clear why a user would want to use the parameter.

Reply:
Done.